app/assets/javascripts/galleries/uploader.js

```diff
@@ -40,6 +42,17 @@ function bindFileInput(fileInput, form, submitButton, formData) {
     imageMaxHeight: 400,
 
     add: function(e, data) {
+      if (typeof limit !== 'undefined' && limit > 1) {
+        var isFirstFile = (fileInput[0].files[0] == data.files[0]);
+        var numUploading = fileInput[0].files.length;
+        if (isFirstFile) numFiles += numUploading;
+        if (numFiles > limit) {
+          if (isFirstFile) alert("You cannot upload more than "+limit+" files at once. Please try again.");
```

Comment:
Can you do string interpolation here or not because ~javascript~

Reply:
Can with template strings, but I think those are ES6 (so we don't yet use that in our codebase).